src/Event/Http/Psr7Bridge.php

```diff
@@ -157,4 +163,31 @@ private static function parseKeyAndInsertValueInArray(array &$array, string $key
 
         $pointer = $value;
     }
+    
+    /**
+     * Parse the username and password from the `Authorization` header.
+     * Only "Basic" is supported.
+     *
+     * @param  array  $headers
+     * @return string[]|null[]
+     */
+    protected static function parseBasicAuthorization(array $headers)
+    {
+        $authorization = trim($headers['authorization'][0] ?? '');
```

Comment:
Just to confirm, at this point all headers are forced to be in lowercase? (not risk of getting `Authorization` instead?)

Reply:
Jup: https://github.com/brefphp/bref/blob/12afd8a49ceb97c2a073b1764ba01cf3efb25e87/src/Event/Http/HttpRequestEvent.php#L298

Reply:
Beautiful thanks!